src/plugins/data/public/ui/filter_bar/filter_item.tsx

```diff
@@ -145,7 +145,14 @@ export function FilterItem(props: Props) {
     const dataTestSubjNegated = filter.meta.negate ? 'filter-negated' : '';
     const dataTestSubjDisabled = `filter-${isDisabled(labelConfig) ? 'disabled' : 'enabled'}`;
     const dataTestSubjPinned = `filter-${isFilterPinned(filter) ? 'pinned' : 'unpinned'}`;
-    return `filter ${dataTestSubjDisabled} ${dataTestSubjKey} ${dataTestSubjValue} ${dataTestSubjPinned} ${dataTestSubjNegated}`;
+    return classNames(
```

Comment:
can we limit the scope of the changes to the Security solution plugin only?

Reply:
yeah I understand, but it has an extra whitespace in the `data-test-subj` and failed the cypress test.
I had a quick fix by adding an extra whitespace `in x-pack/plugins/security_solution/cypress/screens/timeline.ts Line 111` in my previous PR to adapt to it, but I don't think that's a good solution.